ppr-ui/src/assets/styles/layout.scss

```diff
@@ -77,6 +77,10 @@
   }
 }
 
+.col-3, .col-sm-3 {
+  max-width: 25%;
+}
+
```

Comment:
This is actually default by vuetifys grid system.
Why i've explicitly set this though, if we are to move towards standardizing our spacing i believe it might be important to bring visibility to it, if this were a shared file across projects ect.

I've discussed with Tracy the implications, if we stick to our 3-9 column formats for all components, reuse will become much easier. 
To clarify as well, the 3 column is typically our labels/side label column and the 9 col would contain whatever form or content we want to display. 

Seems like a simple thing, but having this consistent means we could adjust the initial label columns and/or form columns in a single place for the vast majority of our apps components.

Reply:
Hmm, `col-3` is usually a third part of row, but 25% is closer to a quoter size.

Reply:
In Vuetify, the grid system consists of 12 Cols.